lib/src/Uncertainty/StatTests/VisualTest.cxx

```diff
@@ -289,7 +282,7 @@ GridLayout VisualTest::DrawPairs(const Sample & sample)
     for (UnsignedInteger j = 0; j < i; ++ j)
     {
       const Indices indices = {j, i};
-      const Cloud cloud(sample.getMarginal(indices), ResourceMap::GetAsString("Drawable-DefaultColor"), ResourceMap::GetAsString("Drawable-DefaultPointStyle"));
+      Cloud cloud(sample.getMarginal(indices));
```

Comment:
A welcome simplification. Since the default colors and point styles are implemented in the `Cloud` class, there is, indeed, no need to get the values from the `ResourceMap` at this point. +1